src/FlyWithLua.cpp

```diff
@@ -5009,6 +5019,130 @@ static int LuaXPLMGetDataRefTypes(lua_State* L)
     }
 }
 
+static int LuaXPLMGetMouseLocationGlobal(lua_State *L)
+{
+    int sGlobalx, sGlobaly;
+    XPLMGetMouseLocationGlobal(&sGlobalx, &sGlobaly);
+
+    lua_pushnumber(L, sGlobalx);
+    lua_pushnumber(L, sGlobaly);
+
+    return 2;
+}
+
+static int LuaXPLMGetScreenSize(lua_State *L)
+{
+    int sWidth, sHeight;
+    XPLMGetScreenSize(&sWidth, &sHeight);
+
+    lua_pushnumber(L, sWidth);
+    lua_pushnumber(L, sHeight);
+
+    return 2;
+}
+
+static int LuaXPLMGetScreenBoundsGlobal(lua_State *L)
+{
+    int sLeft, sTop, sRight, sBottom;
+    XPLMGetScreenBoundsGlobal(&sLeft, &sTop, &sRight, &sBottom);
+
+    lua_pushnumber(L, sLeft);
+    lua_pushnumber(L, sTop);
+    lua_pushnumber(L, sRight);
+    lua_pushnumber(L, sBottom);
+
+    return 4;
+}
+
+static void LuaReceiveMonitorBounds_f(int inMonitorIndex, int inLeft, int inTop, int inRight, int inBottom, void * refcon)
+{
+	auto MonitorBounds = reinterpret_cast<vector<MonitorBoundsStructure>*>(refcon);
+	
+	MonitorBounds->emplace_back(
+		MonitorBoundsStructure{
+			inMonitorIndex,
+			inLeft,
+			inTop,
+			inRight,
+			inBottom
+		}
+	);
+}
+
+static int LuaXPLMGetAllMonitorBoundsOS(lua_State *L)
+{
+	vector<MonitorBoundsStructure> MonitorBounds;
```

Comment:
This should be std::vector

Reply:
I have changed in my branch and will upload once all other changes done.